### PR TITLE
Add checklist of how to structure a pull request

### PR DESCRIPTION
**Pull request by romunov**
Feel free to suggest improvements to this checklist.

### CODE REVIEW

**Comment by jkokosar** [comment]
@romunov There are additional aspects that might be included in PR checklists. Please consider adding them:
* image testing: test build should still be done on a local computer, to validate that the image builds OK (cannot be done from PR)
* image validation: one should attempt to use the test build of the image in a resolwe-bio test (or similar). This is shared responsibility between the one making the PR and the review-er.
* for new image folders, the corresponding image folder/project needs to be created beforehand on AWS ECR for image push to succeed.

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Build Docker image using GitHub Actions
+In order to successfully build a Docker image using GitHub Actions, you will need to:
+* [ ] Prepare all necessary files to build the image (see other examples in this repository)
+* [ ] Create, if not already present, `build.env` file (in the same folder as `Dockerfile`) which includes the following tags:
+
+    * `BASE_IMAGE` is the image used to build your Dockerfile, i.e. `BASE_IMAGE=public.ecr.aws/genialis/resolwebio/base:ubuntu-22.04-14112023`.
+    * `REPOSITORY` is the name of the image once built, for example `REPOSITORY=resolwebio/common`.
+    * `TAG` will be the tag of the image, i.e. `TAG=common:0.0.0`.
```
Comment (AnzeLovse) [inline .github/pull_request_template.md:8]: I would also add a checkbox to stress the importance of updating the tag on existing images.

**Comment by romunov** [comment]
Thank you @jkokosar and @AnzeLovse for the comments. After thinking a bit more about this, I have split the document into two sections. One is about building a Docker image from scratch, the other one is updating the existing image. I believe these two processes are different enough to warrant at least their own section.

**Review by jkokosar — APPROVED** [review]

**Comment by romunov** [comment]
Commits are now squashed. Feel free to merge if there's nothing else.

**Pull request merged**